src/test/fetchCoin.test.js

```diff
@@ -0,0 +1,8 @@
+import fetchSingleCoin from '../services/fetchCoin';
+
+describe('fetchSingleCoin', () => {
+  it('should fetch a single coin data', async () => {
+    const coinData = await fetchSingleCoin('bitcoin');
+    expect(coinData).toBeDefined();
+  });
+});
```

Comment:
Nice work fixing the tests. Both are passing now. However, you seem to have missed some of the changes requested by the previous reviewer (which are also part of the project requirements as shown below)

- [ ] Your app must have unit tests for several components (like Coin.jsx, Navigation.jsx, CoinDetailPage.jsx, HomePage.jsx, App.jsx, etc) of the app and some pure functions in the Redux code (like those in coinSlice and singleCoinSlice)
- [ ] The app should also have integration tests using React Testing Library
  (Student can perform a mocking data fetching and Redux Store connection.)

![Screenshot from 2023-08-26 16-40-07](https://github.com/Peter-Kibet/metrics-app-react-capstone/assets/65233860/f9c64756-8ac2-4e8b-a1aa-232b5ba06b25)